pkg/util/testing/wrappers.go

```diff
@@ -174,6 +174,7 @@ func MakePodSet(name string, count int) *PodSetWrapper {
 			Count: int32(count),
 			Template: corev1.PodTemplateSpec{
 				Spec: corev1.PodSpec{
+					RestartPolicy: "Never",
```

Comment:
nit: why is this changed?

Reply:
When we make job `RestartPolicy = "Never"` by [default](https://github.com/kubernetes-sigs/kueue/blob/main/pkg/util/testingjobs/job/wrappers.go#L47). To compare job and job in podset these two fields should be the same